core/src/entities/cyclonedx/models/license_choice.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Deserialize, Serialize};
+/*
+ * Generated by: https://openapi-generator.tech
+ */
+
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct LicenseChoice {
+    #[serde(rename = "license", skip_serializing_if = "Option::is_none")]
+    pub license: Option<Box<crate::entities::cyclonedx::models::License>>,
```

Comment:
So this is all generated... Is there a point to be adding comments here?  If so, I'd like to see the generic type be shorter.  Could have a "use" statement

Reply:
generated by spec.